### ISSUE TITLE
Implement the On-demand API Deployment, including wiring things up to APIGW

### ISSUE THREAD

**Issue by galenatjpl** [unity-sds/unity-on-demand#27]

**Comment by galenatjpl** [comment]
Most of the work for this issue is done, but we are waiting on SPS/CS to get the final deployment/SSM stuff sorted out, in order to complete the final wiring up of SSM for this issue.  Pushing the remainder of this ticket to 23.3 for completion.

**Comment by galenatjpl** [comment]
I'm going to re-assign this ticket to me, and move it to a "Future Work" (backlog this ticket).
@pymonger @GodwinShen @mike-gangl this is part of the deprecation of the pre-pivot-to-Initiators body of work.
